lib/parse.js

```diff
@@ -73,4 +75,58 @@ module.exports = {
     }
     return resp;
   },
+  customCSS: (file, enc, env) => {
+    // Parse the contents of a file containing CSS output. For any occurences of
+    // `url(...)` with relative paths, store the paths (for later copying) and
+    // replace them with the eventual destination paths.
+    env.customCSSFiles = env.customCSSFiles || [];
+    const src_dir = file.path.substring(0, file.path.lastIndexOf('/'));
+    let css = file.contents.toString(enc);
+    const regex = /url\(([^)]*)\)/g;
+    const matches = [];
+    let nextMatch = regex.exec(css);
+    while (nextMatch !== null) {
+      // Store matches in reverse order for easier reverse-traversing of the css
+      // string while replacing paths.
+      matches.unshift({
+        text: nextMatch[0],
+        index: nextMatch.index,
+      });
+      nextMatch = regex.exec(css);
+    }
+    for (const match of matches) {
+      // Ignore initial `url(` and final `)`
+      let src_path = match.text.substring(4, match.text.length - 1);
+      let start_index = match.index + 4;
+      // Ignore quotes, if content is quoted
+      if (src_path.startsWith("'") || src_path.startsWith('"')) {
+        src_path = src_path.substring(1, src_path.length - 1);
+        start_index = start_index + 1;
+      }
+      if (
+        !src_path.startsWith('http') &&
+        !src_path.startsWith('//') &&
+        !src_path.startsWith('data:')
+      ) {
```

Comment:
@mirisuzanne Do these three cases capture the non-local-path options?

Reply:
Yeah, I think so. That all makes sense to me.